.env

```diff
@@ -1 +1,4 @@
-DATABASE_URL = mongodb+srv://akashgreenwork:atsDashboard123@ats-cluster.jj8g568.mongodb.net/?retryWrites=true&w=majority
\ No newline at end of file
+DATABASE_URL = mongodb+srv://akashgreenwork:atsDashboard123@ats-cluster.jj8g568.mongodb.net/ats-dashboard?retryWrites=true&w=majority
+PORT=3000
+DATABASE_PASSWORD=YVzNTweSbVf1Gq1M
+DATABASE_URL2=mongodb+srv://abhijeet180105:YVzNTweSbVf1Gq1M@cluster0.kgzckrm.mongodb.net/?retryWrites=true&w=majority
```

Comment:
what is the use of the below code and where we are using it
PORT=3000
DATABASE_PASSWORD=YVzNTweSbVf1Gq1M
DATABASE_URL2=mongodb+srv://abhijeet180105:YVzNTweSbVf1Gq1M@cluster0.kgzckrm.mongodb.net/?retryWrites=true&w=majority